internal/slackclient/client.go

```diff
@@ -36,6 +37,21 @@ type Message struct {
 	Type        string
 }
 
+type WriteMessage struct {
+	ThreadTS    string       `json:"thread_ts,omitempty"`
+	Channel     string       `json:"channel"` // required
+	Text        string       `json:"text,omitempty"`
+	Attachments []Attachment `json:"attachments,omitempty"`
```

Comment:
nit: Do we need this for now? If we intend to focus on text for the first cut, then this may be YAGNI. Totally fine to keep it as is too.

Reply:
For now, I've kept it, with an `omitempty` json tag, would revisit this later if/when we think about supporting attachments and blocks (the API also supports blocks - not quite clear on what that means for us yet but its an interesting concept)